content/resources/faculty/contents.lr

```diff
@@ -6,49 +6,50 @@ body:
 
 Below is a _noncomprehensive_ list of homepages for various computer science faculty members, potentially outdated/dead links.
 
-* Adrian Fiech - [afiech](http://www.cs.mun.ca/~afiech/)
-* Andrew Vardy - [av](http://www.cs.mun.ca/~av/)
+Note: some users in this list might be duplicated / not really 'people', it's simply based on ever user within the faculty directory using [this](https://github.com/MUNComputerScienceSociety/Website/blob/master/content/resources/faculty/generate-faculty-page.fish) script.
```

Comment:
Typo: "user" should be "users"

Reply:
702d80f0736974fb003d1c79a730bcde820b3cf5